autoload/lsp/utils/text_edit.vim

```diff
@@ -14,6 +14,24 @@ function! lsp#utils#text_edit#apply_text_edits(uri, text_edits) abort
     endif
 endfunction
 
```

Comment:
add document similar to `lsp#utils#location#_lsp_to_vim_list`

Reply:
Done 😊